src/shogun/machine/gp/Inference.cpp

```diff
@@ -39,18 +39,11 @@
 #include <shogun/mathematics/Math.h>
 #include <shogun/lib/Lock.h>
 
-using namespace shogun;
+#ifdef HAVE_OPENMP
+#include <omp.h>
```

Comment:
why do you need this header?
i might missed it but i dont see anywhere in the code omp API being used, or?